common/djangoapps/util/tests/test_submit_feedback.py

```diff
@@ -44,17 +46,18 @@ def setUp(self):
             username="test",
             profile__name="Test User"
         )
-        # This contains issue_type and course_id to ensure that tags are submitted correctly
```

Comment:
I'm not sure how course_id could ever be present for an anonymous user, but with the changes proposed in this PR, it will only be present for logged in users.

Also, based on the code in `lms/templates/help_modal.html`, it seems like issue_type should always be present, since it is a hidden field that gets populated when the user clicks on a button to view a feedback form.